### PR TITLE
Update Task release note to have v1.6.4

### PR DESCRIPTION
**Pull request by Yiming-Jia**
Update Task release note to have v1.6.4

### CODE REVIEW

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v1.6.4 - June, 19 2019
+* Improve [homepage documentation](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task):
```
Comment (jimmyca15) [inline releaseNotes/AzureDevOpsPipelineExtension.md]: Improve -> Improve**d**

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
...
+  * Add the link the the release note.
```
Comment (jimmyca15) [inline releaseNotes/AzureDevOpsPipelineExtension.md]: Added a link to the release notes.

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 3]
```diff
@@ -2,6 +2,11 @@
...
+  * Add a example of how to consume fetched key-value.
```
Comment (jimmyca15) [inline releaseNotes/AzureDevOpsPipelineExtension.md]: Add -> Add**ed**

a -> a**n**

key-value -> key-value**s**

File: releaseNotes/AzureDevOpsPipelineExtension.md (hunk 1 above)
Comment (Yiming-Jia) [reply releaseNotes/AzureDevOpsPipelineExtension.md]: Fixed

File: releaseNotes/AzureDevOpsPipelineExtension.md (hunk 2 above)
Comment (Yiming-Jia) [reply releaseNotes/AzureDevOpsPipelineExtension.md]: fixed.

File: releaseNotes/AzureDevOpsPipelineExtension.md (hunk 3 above)
Comment (Yiming-Jia) [reply releaseNotes/AzureDevOpsPipelineExtension.md]: Fixed.

File: releaseNotes/AzureDevOpsPipelineExtension.md (hunk 2 above)
Comment (zhenlan) [reply releaseNotes/AzureDevOpsPipelineExtension.md]: Double "the"

Comment (jimmyca15) [reply releaseNotes/AzureDevOpsPipelineExtension.md]: Yeah looks like that change got left out

"the the" -> "to the"

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 4]
```diff
@@ -2,6 +2,11 @@
 
 The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v1.6.4 - June, 19 2019
+* Improved [homepage documentation](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task):
+  * Added a link the release notes.
```
Comment (jimmyca15) [inline releaseNotes/AzureDevOpsPipelineExtension.md]: Still incorrect ?

It should read exactly like this: "Added a link to the release notes"

File: releaseNotes/AzureDevOpsPipelineExtension.md (hunk 2 above)
Comment (Yiming-Jia) [reply releaseNotes/AzureDevOpsPipelineExtension.md]: Yes, fixed.

File: releaseNotes/AzureDevOpsPipelineExtension.md (hunk 4 above)
Comment (Yiming-Jia) [reply releaseNotes/AzureDevOpsPipelineExtension.md]: Fixed.

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 5]
```diff
@@ -2,6 +2,11 @@
 
 The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v1.6.4 - June, 19 2019
+* Improved [homepage documentation](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task):
+  * Added a link to the release notes.
```
Comment (zhenlan) [inline releaseNotes/AzureDevOpsPipelineExtension.md]: Will this avoid some confusion we are referring to some other release notes?
*Added a link to **this** release notes.*

Comment (Yiming-Jia) [reply releaseNotes/AzureDevOpsPipelineExtension.md]: Changed to `* Added a link to this release notes.`

**Pull request merged**